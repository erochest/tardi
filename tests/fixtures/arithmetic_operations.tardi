1 2 +
3 4 -
2 3 *
10 2 /
2.5 1.5 +
3.0 1.5 -
2.0 3.0 *
10.0 2.0 /
5 2.5 +
6 1.5 -
3 2.0 *
10 2.0 /
