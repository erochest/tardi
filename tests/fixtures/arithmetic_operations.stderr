3
-1
6
5
4.0
1.5
6.0
5.0
7.5
4.5
6.0
5.0
